tools/clang/test/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ configure_lit_site_cfg(
   ${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg
   )
 
+configure_lit_site_cfg(
+  ${CMAKE_CURRENT_SOURCE_DIR}/hash_stability/lit.site.cfg.in
+  ${CMAKE_CURRENT_BINARY_DIR}/hash_stability/lit.site.cfg
```

Comment:
Is it necessary to explicitly pass this? Usually lit will find the local site config by the directory it is targeted at.

Reply:
Cannot make it work :(
Would be nice to remove it.

Reply:
This will also create a path for hash_stability to use as temp directory.

Reply:
If you only need it for the temp directory, there are other ways to do that, like CMake's `make_directory` command.

Reply:
Still cannot make it work :(
lit.cfg will fail to find llvm_tools_dir if not load site config.
Tried to load site config of parent folder, but cause other test fail.